### PR TITLE
feature/FOUR-9594:  Add the Close windows

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
Add the Close windows

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-9594

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy

### CODE REVIEW

File: resources/js/tasks/components/TasksPreview.vue [hunk 1]
```diff
@@ -6,15 +6,22 @@
...
-              @click="hide"
-            />
+            <div class="text-right align-items-center">
+              <button
+                type="button"
+                aria-label="Close"
+                class="px-2 py-1 close"
+                @click="hide"
+              >
+                ×
```
Comment (fagubla) [inline resources/js/tasks/components/TasksPreview.vue:22]: use icon

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/tasks/components/TasksPreview.vue (hunk 1 above)
Comment (luNunezProcessmaker) [reply resources/js/tasks/components/TasksPreview.vue:22]: The x icon does not show up, this is the button that appears to be used in many windows

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-82e7a7f370.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-82e7a7f370.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-82e7a7f370.eng.processmaker.net

**Review by fagubla — APPROVED** [review]

**Pull request merged**